crates/re_edit_ui/src/radius.rs

```diff
@@ -0,0 +1,67 @@
+use egui::NumExt as _;
+use re_types::components::Radius;
+use re_viewer_context::ViewerContext;
+
+use crate::response_utils::response_with_changes_of_inner;
+
+pub fn edit_radius_ui(
+    _ctx: &ViewerContext<'_>,
+    ui: &mut egui::Ui,
+    value: &mut Radius,
+) -> egui::Response {
+    let mut abs_value = value.0.abs();
+    let speed = (abs_value * 0.01).at_least(0.001);
+
+    let drag_response = ui.add(
+        egui::DragValue::new(&mut abs_value)
+            .clamp_to_range(false)
```

Comment:
In this particular case we should probably clamp to range. The input is already non-negative (due to the `.abs()`), and letting the user input `-2.0` is not helpful

Reply:
I was pondering that, but didn't want to because of NaN values. You can't debug them if they're not shown here. Tbh not sure what the range clamp would do